src/net/sourceforge/kolmafia/Modifiers.java

```diff
@@ -864,7 +864,6 @@ public void applyPassiveModifiers(final boolean debug) {
                 .filter(SkillDatabase::isPassive)
                 .map(UseSkillRequest::getUnmodifiedInstance)
                 .filter(Objects::nonNull)
```

Comment:
can you also get rid of the mapping to `UseSkillRequest`? It looks like the only point of that was `isEffective`, which is only for G-Lover.

Reply:
Sure, done.